client/src/components/DirectoryAddUpdate.jsx

```diff
@@ -54,6 +56,9 @@ export default function DirectoryAddUpdate() {
   // While loading coop data from ID
   const [loadingCoopData, setLoadingCoopData] = React.useState(false);
 
+  // Alert provider state
+  const [open, close] = useAlert();
```

Comment:
We don't appear to be using the "close" function anywhere.  Do we need it here?  (Like maybe we could just leave it out and have const { open } = useAlert() or whatever the right syntax is.

Reply:
Done!